internal/cli/atlas/livemigration/live_migration.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package livemigration
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/atlas/livemigration/link"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "livemigrations"
```

Comment:
👀  just making sure we want to treat this a single word and not two

Reply:
This was the spec, but I'm happy to change to suit our current best practices. Do we have a naming convention here?

Reply:
again, just double checking, if two words we go camelCase, so `liveMigrations`, then `livemigrations` is also valid, as `live-migrations` and singulars of those, unless we want to stick to `livemigrations` as a single word, no strong preference here

Reply:
Thanks for the clarifications. Let's go with two words (camelCase), as that gives the most natural options. I'll update and do a quick verification it all works as expected.